lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -311,7 +316,12 @@ def __init__(self, config, ctx=None):
 
         self.workspace_url = config.get("workspace-url")
         self.auth_url = config.get("auth-url")
-        self.auth_util = AuthUtil(self.auth_url, self.admin_roles)
+        self.admin_auth_util = AdminAuthUtil(self.auth_url, self.admin_roles)
+        self.user_id = user_id
+        self.token = token
+        self.is_admin = False
+        if self.token:
+            AdminAuthUtil(self.auth_url, self.admin_roles).is_admin(self.token)
```

Comment:
Do we always need to check to see if someone is an admin?

Reply:
Not necessarily. I figure that admins just get superuser powers (can view/modify any job, can run admin functions) that can just skirt around the other auth mechanisms. But we can just put that in the general auth checking stages, too, as necessary.

Reply:
Ok, removed that for now. the `administer` function is the only place that checks admin rights now. I think other changes (converting to using authstrategy for rights checking) will be the next change set.